apps/admin/app/(auth)/register/_components/form-schema.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod";
+
+export const formSchema = z.object({
+  firstName: z
+    .string({
+      required_error: "First name is required",
+    })
+    .min(5, { message: "First name must contain at least 5 character(s)" }),
```

Comment:
```suggestion
    .min(5, { message: "First name must contain at least 5 character(s)" }),
```
why is is required in bracket?

Reply:
@yaju09 If this is about the message in the brackets, then that is because it the way zod wants the error message